user/models.py

```diff
@@ -1,5 +1,16 @@
 from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
 
 
 class User(AbstractUser):
-    pass
+    @staticmethod
+    def validate_password(password: str, error_to_show):
+        if len(password) < 5:
```

Comment:
The `validate_password` function checks if the password length is less than 5. According to the error message, it seems like the password should be at least 5 symbols long. However, the condition `len(password) < 5` will return `True` for a password that is 4 symbols long. So, the condition should be `len(password) < 5` to match the error message.